python/parser.js

```diff
@@ -18,10 +18,12 @@
         if (cs.frames) element += frames(cs.frames)
         if (cs.region) element += region(cs.region, true)
     }
-    if (cs.ignoreRegions) options += ignoreRegions(cs.ignoreRegions)
     if (cs.floatingRegions) options += floatingRegions(cs.floatingRegions)
     if (cs.accessibilityRegions) options += accessibilityRegions(cs.accessibilityRegions)
-    if (cs.layoutRegions) options += layoutRegions(cs.layoutRegions)
+    if (cs.ignoreRegions) options += regions("ignore", cs.ignoreRegions)
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=9697038)
```suggestion
    if (cs.ignoreRegions) {options += regions("ignore", cs.ignoreRegions)}
```